Libs/MRML/Widgets/qMRMLVolumeInfoWidget.cxx

```diff
@@ -186,6 +186,15 @@ void qMRMLVolumeInfoWidget::updateWidgetFromMRML()
     double origin[3] = {0.,0.,0.};
     d->ImageOriginWidget->setCoordinates(origin);
 
+    double IJKToRASDirections[3][3] = { {1,0,0}, {0,1,0}, {0,0,1} };
+    for (int i=0; i<3; i++)
+    {
```

Comment:
I don't like Slicer's current brace indentation style, but for consistency, please use that (indent the `{` by two spaces).